src/controllers/controller.js

```diff
@@ -0,0 +1,62 @@
+import User from '../models/model';
+
+class UsersController {
+  constructor() {
+    this.users = [];
+  }
+
+  createUser(login, password, age) {
+    const newUser = new User(login, password, age);
```

Comment:
Can't create user with this payload:
```{
	"login": "Test1212asd3",
	"password": "asdas2e12312",
	"age": 22
}```

Reply:
If unable to create a user - should return a JSON error (here and everywhere)

Reply:
Updated -- for the first note, are you sending that payload in x-www-form-urlencoded?

I've added JSON error messaging throughout.